public/js/components/Scopes.js

```diff
@@ -71,15 +73,99 @@ function getScopes(pauseInfo) {
       scopes.push([
         scope.getIn(["object", "class"]),
         fromJS({
+          isScope: true,
           value: scope.get("object")
         })
       ]);
     }
+    index++;
   } while (scope = scope.get("parent")); // eslint-disable-line no-cond-assign
 
   return scopes;
 }
 
+function isObject(obj) {
+  return obj.getIn(["value", "type"]) === "object";
+}
+
+function loadedProperties(obj, loadedObjects) {
+  return loadedObjects.get(obj.getIn(["value", "actor"])) ||
+         obj.get("prefetchedProperties");
+}
+
+function sortProperties(a, b) {
+  const obj = isObject(a[1]) && !isObject(b[1]) ? -10 : 10;
+  const alpha = a[0] < b[0] ? -1 : 1
+
+  return obj + alpha > 0 ? -1 : 1;
```

Comment:
I pushed the objects to the bottom of the list because aesthetically it looked better to me, but happy to revert that that change if it seems weird.

![screen shot 2016-06-24 at 3 02 03 pm](https://cloud.githubusercontent.com/assets/254562/16347805/a40a865a-3a1c-11e6-9034-5ad52f2b3168.png)
![screen shot 2016-06-24 at 3 01 52 pm](https://cloud.githubusercontent.com/assets/254562/16347806/a40de340-3a1c-11e6-8b4c-464679d854c5.png)